hawk-spring/src/main/java/so/hawk/java/handler/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,39 @@
+package so.hawk.java.handler;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
```

Comment:
exception handler itself should not depend on "http" module. it should just call some callback when exception is occurred.